bin/docker-build.js

```diff
@@ -860,9 +1043,16 @@ function importImageSnapshotsIntoImgapi(builder, opts, callback) {
            }
         ], function importCleanup(err) {
             if (err) {
-                log.error('Error importing images into IMGAPI', err);
+                log.error('Unable to import image into imgapi: %s', err);
                 // Remove this image if it failed to import/validate.
+                if (!imageSdcDocker) {
+                    cb(err);
+                    return;
+                }
                 var image_uuid = imageSdcDocker.image_uuid;
+                log.error('Unable to import image %s into imgapi: %s',
```

Comment:
No way - that one is totally different ;)

I had meant to use that for the case where there was imageSdcDocker (and thus no image_uuid) - I'll move that log into the `if (!imageSdcDocker) {` block.